Added stubs for all known function calls, moved state objects, stores as hash table.

Ragnarhawk opened the pull request:
First re-write with an independent object space defined in state.  Defining the stores list as a hash table (Map)

Ragnarhawk committed:
Updating shop.js - Hawk's initial

This commit has functions stubbed out, and will now allow store creation, but has no functionality in the stores.
With this version, I have created a new space under state for Julexar's code to use, and a sub space under that for the Item Store

Signed-off-by: Ragnarhawk <ragnarhawk@gmail.com>

Julexar committed:
Update milestone.yml (#15)

Ragnarhawk commented:
First re-write
* Independent object space defined in state. (state.julexar contains state.julexar.shopObjs)
* Defining the stores list as a hash table (Map)
* Created stubs for all known functions
* Function createStore now working correctly

Julexar reviewed (changes requested):
Please use `${function.name}` instead of a constant

Julexar committed:
Merge branch 'rewrite-item-store' into hawk's-updated-rewrite

Ragnarhawk committed:
Updated shop.js to address CR to use ${} 

Updated shop.js to address CR to use ${} in sendChats with variables, and updated sendChats to use backticks instead of quotes for their body

Signed-off-by: Ragnarhawk <ragnarhawk@gmail.com>

Julexar reviewed (changes requested):
Line 2169, 2176, 2970, 2972, 3013, 3049, 3091, 3121, 3126: Revert to using "

Ragnarhawk committed:
Reverting strings with no variables to use "

Addressing comments on CR

Signed-off-by: Ragnarhawk <ragnarhawk@gmail.com>
